src/components/ErrorDecoder.tsx

```diff
@@ -0,0 +1,94 @@
+import {useMemo} from 'react';
+import {useRouter} from 'next/router';
+
+function replaceArgs(
+  msg: string,
+  argList: Array<string | undefined>,
+  replacer = '[missing argument]'
+): string {
+  let argIdx = 0;
+  return msg.replace(/%s/g, function () {
+    const arg = argList[argIdx++];
+    // arg can be an empty string: ?invariant=377&args[0]=&args[1]=count
+    return arg === undefined || arg === '' ? replacer : arg;
+  });
+}
+
+// When the message contains a URL (like https://fb.me/react-refs-must-have-owner),
+// make it a clickable link.
```

Comment:
Can you add an example of this to the PR description?

Reply:
I can't find this kind of error message in the recent version of React. This kind of error message might only exist in the earlier version of React.

Reply:
@rickhanlonii Found an example: https://react-dev-git-fork-sukkaw-error-decoder-fbopensource.vercel.app/errors/119

The PR description has also been updated.